mm2src/adex_cli/src/tests/mod.rs

```diff
@@ -142,13 +142,14 @@ async fn test_enable() {
 }
 
 async fn test_activation_scheme() {
+    let _ = std::fs::remove_file(get_activation_scheme_path().unwrap());
     init_activation_scheme().await.unwrap();
     let scheme = get_activation_scheme().unwrap();
     let kmd_scheme = scheme.get_activation_method("KMD");
     assert!(kmd_scheme.is_some());
     let kmd_scheme = kmd_scheme.unwrap();
     assert_eq!(kmd_scheme.get("method").unwrap().as_str().unwrap(), "electrum");
-    assert_eq!(kmd_scheme.get("servers").unwrap().as_array().unwrap().iter().count(), 3);
+    assert_ne!(kmd_scheme.get("servers").unwrap().as_array().unwrap().iter().count(), 0);
```

Comment:
Im a little out of scope of your feature. why servers shouldn't be equal 0 here after removing activation scheme?

Reply:
there is `assert_ne` )